src/frontend/src/views/Data/components/SamplesView/components/SamplesTable/index.tsx

```diff
@@ -72,24 +88,48 @@ const columns: ColumnDef<Sample, any>[] = [
 
       const onChange = getToggleAllRowsSelectedHandler();
 
-      return <Checkbox stage={checkboxStage} onChange={onChange} />;
+      return (
+        <CellComponent>
+          <Checkbox stage={checkboxStage} onChange={onChange} />
+        </CellComponent>
+      );
     },
     cell: ({ row }) => {
       const { getIsSelected, getToggleSelectedHandler } = row;
 
       const checkboxStage = getIsSelected() ? "checked" : "unchecked";
       const onChange = getToggleSelectedHandler();
 
-      return <Checkbox stage={checkboxStage} onChange={onChange} />;
+      return (
+        <CellComponent>
+          <Checkbox stage={checkboxStage} onChange={onChange} />
+        </CellComponent>
+      );
     },
   },
   {
     id: "privateId",
     accessorKey: "privateId",
+    minSize: 350,
     header: ({ header }) => (
       <SortableHeader header={header}>Private ID</SortableHeader>
     ),
-    cell: ({ getValue }) => <CellBasic primaryText={getValue()} />,
+    cell: ({ getValue, row }) => {
+      const uploader = row?.original?.uploadedBy.name;
+      return (
+        <StyledPrivateId
+          primaryText={getValue()}
+          secondaryText={uploader}
+          shouldTextWrap
+          primaryTextWrapLineCount={1}
```

Comment:
What exactly does `primaryTextWrapLineCount` this mean? After 1 line of wrapped text it truncates? This is a Janeece question and maybe something you've already talked about this with her, but it seems like this could cause problems if users can't always be guaranteed to be able to see the entirety of the private id value. (That said, if we have user re-sizable columns -- I forget if that's being set up with the refactor -- it's not so big a deal since if they really do want to read their exact private id, they could widen the column as needed.)

Reply:
It wraps up to x number of lines, and then it truncates with an ellipsis. However, the id column has a tooltip on hover that the user can use to view the entire ID even if the column is truncated